src/main/java/dinkplugin/notifiers/LootNotifier.java

```diff
@@ -316,6 +316,7 @@ private void handleNotify(Collection<ItemStack> items, String dropper, LootRecor
                 .replacement("%LOOT%", lootMsg)
                 .replacement("%TOTAL_VALUE%", Replacements.ofText(QuantityFormatter.quantityToStackSize(totalStackValue)))
                 .replacement("%SOURCE%", source)
+                .replacement("%COUNT%", Replacements.ofText(kc != null ? kc.toString() : "unknown"))
```

Comment:
Document this in the readme too

something like
```
`%COUNT%` will be replaced with the source kill count, or unknown if it could not be calculated
```